factory/mock/storageManagerStub.go

```diff
@@ -75,18 +53,12 @@ func (sms *StorageManagerStub) IsPruningEnabled() bool {
 	return false
 }
 
-// EnterSnapshotMode --
-func (sms *StorageManagerStub) EnterSnapshotMode() {
-	if sms.EnterSnapshotModeCalled != nil {
-		sms.EnterSnapshotModeCalled()
-	}
-}
-
-// ExitSnapshotMode --
-func (sms *StorageManagerStub) ExitSnapshotMode() {
-	if sms.ExitSnapshotModeCalled != nil {
-		sms.ExitSnapshotModeCalled()
+// IsPruningBlocked --
```

Comment:
maybe remove all occurrences of double `-` found in this file?

Reply:
Done.